index.html

```diff
@@ -1,87 +1,108 @@
+                <img src="src/images/art/Quartzshaper Editted.webp" alt="Quartzshaper Editted Art">
+                <img src="src/images/art/osu! Halloween 2022 Fanart.webp" alt="osu! Halloween 2022 Fanart">
+                <img src="src/images/art/Blue Houraisan Kaguya.webp" alt="Blue Houraisan Kaguya Art">
+                <img src="src/images/art/3939.webp" alt="3939 Art">
+                <img src="src/images/art/Wishful Wake.webp" alt="Wishful Wake Art">
+                <img src="src/images/art/osu! Winter 2023 Fanart.webp" alt="osu! Winter 2023 Fanart">
+                <img src="src/images/art/osu! Beach Episode Fanart.webp" alt="osu! Beach Episode Fanart">
+                <img src="src/images/art/osu! Halloween 2023 Fanart.webp" alt="osu! Halloween 2023 Fanart">
+                <img src="src/images/art/osu! Beatmap World Fanart.webp" alt="osu! Beatmap World Fanart">
+        </div>
+        <button class="arrow" id="right-arrow">&#9654;</button>
+        </div>
+        <div class="button-container">
+            <a href="./illustrations" class="view-gallery">View Gallery</a>
         </div>
-      </div>
-    </section>
-
-    <section class="nav-bar">
-      <h2><a href="#home" class="nav-link">Home</a></h2>
-      <h2><a href="#portfolio" class="nav-link">Portfolio</a></h2>
-      <h2><a href="miku.html" class="nav-link">Miku</a></h2>
     </section>
-
-    <section class="gallery-section" id="portfolio">
-      <div class="image-grid">
-        <img src="resources/art/Quartzshaper Editted.webp" alt="Quartzshaper Editted" />
-        <img src="resources/art/osu! Halloween 2022 Fanart.webp" alt="osu! Halloween 2022 Fanart" />
-        <img src="resources/art/Blue Houraisan Kaguya.webp" alt="Blue Houraisan Kaguya" />
-        <img src="resources/art/3939.webp" alt="3939" />
-        <img src="resources/art/Wishful Wake.webp" alt="Wishful Wake" />
-        <img src="resources/art/osu! Winter 2023 Fanart.webp" alt="osu! Winter 2023 Fanart" />
-        <img id="censored" src="resources/art/osu! Beach Episode Fanart Censored.webp" alt="osu! Beach Episode Fanart" />
-        <img src="resources/art/osu! Halloween 2023 Fanart.webp" alt="osu! Halloween 2023 Fanart" />
-        <img src="resources/art/osu! Beatmap World Fanart.webp" alt="osu! Beatmap World Fanart" />
-      </div>
+    <section id="info-box-container">
+        <div id="info-box">
+            <h2>This site is a work in progress.</h2>
+            <p>So as a joke, I went to my friend's house wearing Pekora's wig and clothes. <br>I could barely stop my
+                laughter
+                as he went
+                as red as a tomato and looked at me from head to toe with a bit of drool in his mouth. <br>The way he
+                stared
+                made mde feel a
+                bit funny too, but I decided to tease him more by taking off my clothes. <br>He asked me, "Are you
+                serious?"
+                and
+                I said "Yep
```

Comment:
LOL i feel like this is gonna get me flagged for something is there a less nsfw copypasta